Probesys/bileto
# imp: Setup Atkinson Hyperlegible font

Changes proposed in this pull request:

- Change the default font for Atkinson Hyperlegible
- Change -apple-system by -system-ui
- Use percentage in HTML font-size

How to test the feature manually:

1. Open Bileto
2. Check it uses Atkinson Hyperlegible font

Pull request checklist:

- [x] locales are synchronized N/A

_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
